app/src/main/java/com/github/geohunt/app/ui/components/tutorial/WelcomeScreen.kt

```diff
@@ -0,0 +1,114 @@
+package com.github.geohunt.app.ui.components.tutorial
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+/**
+ * The welcome displayed when the user opens the application
+ * for the first time
+ *
+ * @param shouldShowTutorial The state of whether the tutorial should be shown
+ */
+@Composable
+fun WelcomeScreen(shouldShowTutorial: MutableState<Boolean>){
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top,
+    ) {
+        // The welcome text of the application
+        Text(
+            text = "GeoHunt",
+            textAlign = TextAlign.Center,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 40.dp, start = 10.dp, end = 10.dp)
+                .testTag("Welcome Label"),
+            fontSize = 80.sp,
+            fontStyle = FontStyle.Italic,
+            fontFamily = FontFamily.Cursive,
+            fontWeight = FontWeight.Bold,
```

Comment:
To stay coherent with the rest of the app you should use the Lobster font.

Reply:
That's done.